### ISSUE TITLE
Issue template not working 

### ISSUE THREAD

**Issue by Subhangini** [prathimacode-hub/ML-ProjectKart#55]
I saw the repo and found that there is an issue template but it's not working when I am here to create an issue so I want to solve this problem. 
Please assign this issue to me.

**Comment by prathimacode-hub** [comment]
Issue template is .md file, not a .yml file (.yml file is interactive but .md file is only to take this as a template). 
.md file is always like a README file. When you fork the project, you can copy the template and raise an issue using the template given. Issues assigning would be starting from 10th June, from 10AM onwards. I kindly request you to raise an issue by then if you like to work on your own issue, else list of issues are already created. It will be alloted on first come first serve basis, but would count if interest is expressed only after stipulated time. This is to give fair chance for eveyone to contribute. @Subhangini

**Comment by prathimacode-hub** [comment]
If you want to work on this issue, you can. Always welcome to make it interactive, but this won't come under counting towards LGM-SOC. Is this ok for you? @Subhangini

**Comment by Subhangini** [comment]
> Issue template is .md file, not a .yml file (.yml file is interactive but .md file is only to take this as a template).
> .md file is always like a README file. When you fork the project, you can copy the template and raise an issue using the template given. Issues assigning would be starting from 10th June, from 10AM onwards. I kindly request you to raise an issue by then if you like to work on your own issue, else list of issues are already created. It will be alloted on first come first serve basis, but would count if interest is expressed only after stipulated time. This is to give fair chance for eveyone to contribute. @Subhangini

No, I have previously created Issue Template in .md file extension and they appear whenever we create a new issue as a template we don't need to copy them

**Comment by prathimacode-hub** [comment]
Ok, that's great. Do you want to do? But this won't be counted under LGMSOC. Is it ok? @Subhangini

**Comment by Subhangini** [comment]
> Ok, that's great. Do you want to do? But this won't be counted under LGMSOC. Is it ok? @Subhangini

but why i will be working on this issue after june 10 ? So it should be counted?

**Comment by prathimacode-hub** [comment]
I agree to you point since it's after june 10th. So you want this to count under LGMSOC? @Subhangini

**Comment by prathimacode-hub** [comment]
Fine, I shall get this into count now using label, later if this label isn't counted on leaderboard, I shouldn't be blamed. Agree? @Subhangini

**Comment by prathimacode-hub** [comment]
Glad to see your initiation and enthusiasm to make things work. Kudos to you. @Subhangini

**Comment by Subhangini** [comment]
> Right, I shall get this into count now using label, later if this label isn't counted on leaderboard later, I shouldn't be blamed. Agree? @Subhangini

Is there any instruction on labels?

**Comment by prathimacode-hub** [comment]
> > Right, I shall get this into count now using label, later if this label isn't counted on leaderboard later, I shouldn't be blamed. Agree? @Subhangini
> 
> Is there any instruction on labels?

Yes, there is. I shall assign this issue to you. What say? @Subhangini

**Comment by Subhangini** [comment]
>Yes, there is. I shall assign this issue to you. What say? @Subhangini

How much I know, I think there are four labels `level0` ` level1` ` level2 ` `level3` and i think if you will assign any one of these they will be counted

**Comment by prathimacode-hub** [comment]
> > Yes, there is. I shall assign this issue to you. What say? @Subhangini
> 
> How much I know, I think there are four labels `level0` ` level1` `level2` `level3` and i think if you will assign any one of these they will be counted

Here, it's level 1 - 4. This would count under level 1.

**Comment by Subhangini** [comment]
Ya sure

**Comment by prathimacode-hub** [comment]
> Ya sure

Assigned

**Comment by Subhangini** [comment]
> Assigned
I will work on it  soon. Thank you

**Comment by prathimacode-hub** [comment]
You hadn't updated anything since past 3 days. @Subhangini

**Comment by Subhangini** [comment]
> You hadn't updated anything since past 3 days. @Subhangini

sorry for the delay, I will do it(PR) today
